build.sbt

```diff
@@ -441,6 +441,23 @@ lazy val notificationworkerlambda = lambda("notificationworkerlambda", "notifica
     riffRaffArtifactResources += (file("cdk/cdk.out/RegistrationsDbProxy-PROD.template.json"), s"registrations-db-proxy-cfn/RegistrationsDbProxy-PROD.template.json")
 )
 
+lazy val ec2SenderWorker = Project("sender-worker", file("senderworker"))
+  .dependsOn(common, notificationworkerlambda, commontest % "test->test")
```

Comment:
If my understanding is correct, if we depend on notificationworkerlambda then we can access the classes defined in that project inside the `sender-worker` project?

Reply:
Yes, that's right.  Again there are more work needed to be able to call `notificationworkerlambda` - the SSM configuration things.  It will be handled in LIVE-4736.